src/porepy/viz/exporter.py

```diff
@@ -112,6 +112,7 @@ def __init__(
         grid: Union[pp.Grid, pp.MixedDimensionalGrid],
         file_name: str,
         folder_name: Optional[str] = None,
+        length_scale: float = 1.0,
```

Comment:
Maybe I have gotten lost in the mixins, but should we (is it possible to) initialize model.exporter with the length scale? I'm not sure if things are composed so that this is possible, e.g., if the scaling is set sufficiently early.

Reply:
Yes, this happens in initialize_data_saving of DataSavingMixin